src/Agent.Worker/ExecutionContext.cs

```diff
@@ -19,13 +19,14 @@ public interface IExecutionContext : IAgentService
         TaskResult? Result { get; set; }
         TaskResult? CommandResult { get; set; }
         CancellationToken CancellationToken { get; }
+        Int32 TimeoutInMinutes { get; }
```

Comment:
Why restrict to Minutes, would timespan better? like i want to timeout in seconds.